templates/rustdoc/topbar.html

```diff
@@ -84,6 +84,14 @@
                                 </a>
                             </li>
 
+                            {# A link to the release's source view #}
+                            <li class="pure-menu-item">
+                                <a href="{{ crate_url | safe }}/source/" title="Browse source of {{ metadata.name }}-{{ metadata.version }}" class="pure-menu-link">
+                                    {{ "folder-open" | fas(fw=true) }}
```

Comment:
I'm curious about this: why change `far` to `fas(fw=true)`? What does it change?

Reply:
It changes the icon style. Since the "Source" menu item is being relocated to the menu, the original icon didn't match its new siblings in fill or alignment. By switching to `fas`, it's now a solid filled icon. By adding `fw=true`, it's set to be full-width so it is visually aligned with the icons above and below.

Here's a screenshot of the final output to demonstrate the consistency.
![Screen Shot 2020-12-28 at 21 54 14](https://user-images.githubusercontent.com/139487/103358628-0ed65c80-4a84-11eb-8222-3605790b2fdb.png)

Reply:
Nice!